### DISCUSSION TITLE
is there any way to use OR

### DISCUSSION THREAD

**Discussion by ijanggmu** [Biarity/Sieve#195] [General]
username@=test OR FullName @=test ?

**Comment by caiosecco** [comment]
Looking for the same... I don't think so.
At least I also didn't figure it out.

**Reply by jon-armen** [reply]
@ijanggmu , there is.  Filter=(username|FullName)@=test

**Reply by sathishsuresh04** [reply]
@jon-armen  it works well with the properties configured via SievePropertyMapper. However, I've encountered an issue when combining SievePropertyMapper properties with custom filters. Instead of applying an OR logic, it's applying AND. Have you come across this or is there potentially something I'm overlooking?

**Comment by CodeWrapper12** [comment]
@ijanggmu You can try like this 
username | FullName @=test
